src/elements/OCDrop.vue

```diff
@@ -1,12 +1,15 @@
 <template>
-  <div :uk-drop="$_ocDrop_props">
+  <div :id="$_ocDrop_id" :uk-drop="$_ocDrop_props" @click="$_ocDrop_close">
```

Comment:
I'd really love to see this more in a vue.js manner to be implemented:
- drop is visible/hidden using v-if or v-show
- clicking the button changes the state upon the drop is shown

We need to get rid of this uikit magic .. one day ...

Reply:
Agree.